mgo/find.go

```diff
@@ -63,6 +63,9 @@ type (
 		// Whether or not to include total count of documents matching filter in the cursor
 		// Specifying true makes an additionnal query
 		CountTotal bool
+		// The names of multiple fields being paginated and sorted on
+		PaginatedFields []string
+		SortOrders      []int
```

Comment:
Could you add a description to SortOrder?

Maybe also mention that PaginatedFields takes precedence to PaginatedField?